client/app/meal/meal.html

```diff
@@ -1,91 +1,166 @@
+<link rel="stylesheet" href="../../styles/normalize.css">
```

Comment:
I think we should put this one in the index.html. I personally forgot to include it, but I think its best practice for us to all use it.